### PR TITLE
IMAGEDAM-1484 - Add label 'Collections' to icon in Navigation

### PR DESCRIPTION
**Pull request by Conalb97**
## What does this change?

To raise visibility and awareness of Collections, the word 'Collections' has been added next to the Collections icon in the navigation. The behaviour of this label must match the existing labels and as such is resizeable, so the text is removed when the user shrinks the page.

image
image

## How should a reviewer test this change?

These changes can be tested by confirming that when landing on BBC Images tester can clearly see the link to Collections in the navigation, in the same style as 'Show preview' and 'Show info panel' buttons. Clicking on the icon or the word 'Collections' opens up the Collections panel.

On smaller breakpoints where there isn't room for the full button just the icon should display - consistent with 'Show preview' and 'Show info panel' buttons which just display as icons.

## Who should look at this?
@guardian/digital-cms

## Tested? Documented?
- [x] locally by committer
- [ ] locally by Guardian reviewer
- [ ] on the Guardian's TEST environment
- [ ] relevant documentation added or amended (if needed)

### CODE REVIEW

**Review by AndyKilmory — COMMENTED** [review]
All looks good to me - can approve if required

**Comment by paperboyo** [comment]
Hi, Thanks for that. Eminently sensible, I think.

One tiny thing: I think we should now remove the duplicated/redundant now second instance of “Collections” (highlighted below):

image

File: kahuna/public/js/components/gr-panel-button/gr-panel-button-small.html [hunk 1]
```diff
@@ -11,9 +11,10 @@
             gr-track-click-data="ctrl.trackingData('Hide panel')"
             aria-label="Hide {{ctrl.name}} panel">
         <gr-icon>{{ctrl.icon}}</gr-icon>
```
Comment (twrichards) [inline kahuna/public/js/components/gr-panel-button/gr-panel-button-small.html:13]:
```suggestion
        <gr-icon style="margin-bottom: 2.5px;">{{ctrl.icon}}</gr-icon>
```
minor thing but I think this looks better...
| Before | After |
| --- | --- |
| image | image |

**Review by twrichards — APPROVED** [review]
@paperboyo and I have tested this on `TEST` - one minor suggestion for better alignment of the collections icon w.r.t. to the new `Collections` text - assuming you agree please 'accept the suggestion' and we'll merge

**Pull request merged**